src/components/search/index.tsx

```diff
@@ -37,12 +40,17 @@ export const SearchInput: React.FC = () => {
   }, []);
 
   return (
-    <StyledSearchInput>
-      <Input
-        onChange={debouncedHandleChange}
-        type="text"
-        placeholder="Search Shoply"
-      />
-    </StyledSearchInput>
+    <>
+      <StyledSearchInput>
+        <Input
+          onChange={debouncedHandleChange}
+          type="text"
+          placeholder="Search Shoply"
+        />
+      </StyledSearchInput>
+      {products?.map((product) => (
```

Comment:
I think this is not correct. Because you're creating multiple lists of one item as you iterate over products

Reply:
Hmm Okay i see, what you think is another way to tackle it?